.github/workflows/function-composition-bench.yml

```diff
@@ -140,22 +144,20 @@ jobs:
         KUBECONFIG=/etc/kubernetes/admin.conf kubectl get -n chained-functions-eventing ksvc
         KUBECONFIG=/etc/kubernetes/admin.conf kubectl get -n chained-functions-eventing trigger
 
-    - name: Start Experiment
-      run: ./bin/grpcurl -plaintext 10.96.0.84:90 -d @ Timeseries.StartExperiment < ./function-images/tests/chained-function-eventing/ts-experiment.json
-
-    - name: Invoke Thrice
+    - name: Invoke
       run: |
-        ./bin/grpcurl -d '{\"name\": \"Mert\"}'  -plaintext producer.chained-functions-eventing.192.168.1.240.sslip.io:80 helloworld.Greeter.SayHello
```

Comment:
FYI, I also use

```bash
test $(wc -l < rps10.00_lat.csv) -eq 50
```

to compare the actual number of latencies to expected, as the invoker will silently drop failed eventing invocations (instead of writing 0).